content/zh/docs/helm/helm_install.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Helm Install"
```

Comment:
In some cases you translate the title and in some cases you do now. Either is fine, but it will just change what shows on https://helm.sh/zh/docs/helm/.

Reply:
OK，I'll improve it.